lldb/source/Commands/CommandObjectTarget.cpp

```diff
@@ -1506,13 +1513,39 @@ static bool LookupAddressInModule(CommandInterpreter &interpreter, Stream &strm,
 
     ExecutionContextScope *exe_scope =
         interpreter.GetExecutionContext().GetBestExecutionContextScope();
-    DumpAddress(exe_scope, so_addr, verbose, all_ranges, strm);
+    DumpAddress(exe_scope, so_addr, verbose, all_ranges, strm, nullptr);
```

Comment:
So the point of the default argument is that you don't need to pass a nullptr here. If you don't pass anything, it defaults to nullptr.

This means all existing calls stay the same and only paths where your matching took place would pass an argument here.